test/cypress/integration/group-proposals.spec.js

```diff
@@ -286,7 +277,7 @@ describe('Proposals - Add members', () => {
     })
   })
 
-  it(`proposal-based invitation link has ${groupInviteLinkExpiry.proposal} days of expiry`, () => {
+  it(`invitee can not join the group with the expired invitation link`, () => {
```

Comment:
This may give a linter error as backticks should only be used when there's string interpolation. (e.g. ` needs to be changed to ')

Reply:
Done.